client/App.tsx

```diff
@@ -17,9 +17,17 @@ function App() {
         if (mutation.attributeName === "class") {
           const isDarkMode = body.classList.contains("graphiql-dark");
           if (isDarkMode) {
-            document.documentElement.style.setProperty("--app-background-color", "hsl(219, 29%, 18%)");
+            document.documentElement.style.setProperty(
+              "--app-background-color",
+              "hsl(219, 29%, 18%)"
+            );
+            localStorage.setItem("background-color", "dark");
           } else {
-            document.documentElement.style.setProperty("--app-background-color", "hsl(219, 28%, 100%)");
+            document.documentElement.style.setProperty(
+              "--app-background-color",
+              "hsl(219, 28%, 100%)"
+            );
+            localStorage.setItem("background-color", "light");
```

Comment:
can we change navbar light mode background color to be #bdb8d6
and the navbar lightmode box shadow to be #887eb6